packages/front-end/components/OpenVisualEditorLink.tsx

```diff
@@ -98,6 +104,38 @@ const OpenVisualEditorLink: FC<{
 
       window.location.href = url;
```

Comment:
Does this need a return statement here?  I'm pretty sure javascript execution continues after changing the href

Reply:
🤷🏾 My assumption was no but I can add for safety

Reply:
Added